modules/document_repository/php/files.class.inc

```diff
@@ -65,87 +48,64 @@ class Files extends \NDB_Page
      *
      * @return ResponseInterface The outgoing PSR7 response
      */
-    public function handle(ServerRequestInterface $request) : ResponseInterface
+    public function handle(ServerRequestInterface $request): ResponseInterface
     {
         $config = \NDB_Factory::singleton()->config();
         $path   = \Utility::appendForwardSlash(
             $config->getSetting("documentRepositoryPath")
```

Comment:
Hi @johnsaigle, the README.md has `docRepoPath` instead of `documentRepositoryPath` for the path.

Reply:
Thanks, good catch